apps/site/lib/site_web/controllers/customer_support_controller.ex

```diff
@@ -167,8 +167,12 @@ defmodule SiteWeb.CustomerSupportController do
   @spec validate_recaptcha(map) :: :ok | String.t()
   defp validate_recaptcha(%{"recaptcha_response" => response}) do
     case Recaptcha.verify(response) do
-      {:ok, _} -> :ok
-      {:error, [error]} when error in @expected_recaptcha_errors -> "recaptcha"
+      {:ok, _} ->
+        :ok
+
+      {:error, [error]} when error in @expected_recaptcha_errors ->
+        _ = Logger.warn("reCAPTCHA failed validation: #{error}")
```

Comment:
@digitalcora let's reword the error to `"recaptcha failed_validation=#{error}"`, as that will expedite certain analyses in Splunk